docs/05-reference/api/create-stream.mdx

```diff
@@ -67,3 +68,72 @@ https://livepeer.studio/api/stream
     {other asset object keys}
 }
 ```
+
+## Add access control policy to stream
+
+Create a stream with a playback policy.
+
+A `playbackPolicy` type can be either:
+- public - No access control is applied to the asset
+- jwt - A JWT is required to playback the asset
+
+If you set your stream or asset's playback policy to JWT, you 
+will need to sign a token with a signing key and put it in the 
+playback URL to be able to playback your stream.
+
+### Request
+
+```bash
+curl -X POST 'https://livepeer.studio/api/stream' \
+-H 'content-type: application/json' \
+-H 'authorization: Bearer {api_key}' \
+-d '{
+  "name": "test_stream",
+  "profiles": [
+    {
+      "name": "720p",
+      "bitrate": 2000000,
+      "fps": 30,
+      "width": 1280,
+      "height": 720
+    },
+    {
+      "name": "480p",
+      "bitrate": 1000000,
+      "fps": 30,
+      "width": 854,
+      "height": 480
+    },
+    {
+      "name": "360p",
+      "bitrate": 500000,
+      "fps": 30,
+      "width": 640,
+      "height": 360
+    }
+  ],
```

Comment:
We can omit these profiles as they're not required. Will make it clearer about what we are showing here.